opencog/nlp/learn/link-pipeline.scm

```diff
@@ -128,12 +128,7 @@
 ; it parsed, and then updates the counts for the observed words and word
 ; pairs.
 (define (observe-text plain-text)
-	(begin
-		(relex-parse plain-text)  ;; send plain-text to server
-		(update-link-counts (get-new-parsed-sentences))
-		(release-new-parsed-sents)
-		(delete-sentences)
-	)
+	(update-link-counts (nlp-parse plain-text))
```

Comment:
Please revert this. This change almoost surely breaks langauge learning.  You are making teh faulty assumption that learning uses Englsih .. it doesn't, it uses the ANY and AMY langauges, which do not have either relex nor r2l rules in them.  There's no relex because .. its not english.